components/rhs_thread/rhs_thread.jsx

```diff
@@ -404,6 +406,23 @@ export default class RhsThread extends React.Component {
             );
         }
 
+        const channel = ChannelStore.get(selected.channel_id);
```

Comment:
We shouldn't add any new uses of the ChannelStore. If you need a channel you should pull it in from a selector [in the container](https://github.com/mattermost/mattermost-webapp/blob/master/components/rhs_thread/index.js) and add it to the component's props